### PR TITLE
do not hide cookie jar problems

### PR DESCRIPTION
**Pull request by jrwren**

### CODE REVIEW

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-cge.jujugui.org:8080//job/charmstore-client/172/

**Comment by jrwren** [comment]
previous error message:

```
ERROR cannot create the charm store client: cannot load the cookie jar
```

New error message:

```
ERROR cannot create the charm store client: cannot load cookies: open /Users/jrwren/.go-cookies: permission denied
```

Users will now have a clue what file is causing the problem and may be knowledgable enough to chown or chmod it themselves.

**Comment by jrwren** [comment]
Addresses https://github.com/juju/charmstore-client/issues/109

**Review by fabricematrat — APPROVED** [review]

File: cmd/charm/charmcmd/cmd.go [hunk 1]
```diff
@@ -127,7 +127,7 @@ func newCharmStoreClient(ctxt *cmd.Context, auth authInfo, channel params.Channe
 		Filename:         cookiejar.DefaultCookieFile(),
 	})
 	if err != nil {
-		return nil, errgo.New("cannot load the cookie jar")
+		return nil, err
```
Comment (bac) [inline cmd/charm/charmcmd/cmd.go:130]: It's nice when doing the best thing for the user is also much less work.

**Review by bac — APPROVED** [review]

**Comment by jrwren** [comment]
:shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci-gce.jujugui.org:8080/job/charmstore-client-merge

**Pull request merged**